db/api.sql

```diff
@@ -84,6 +84,7 @@ create table suppression_files (
 create table suppressions (
     id serial primary key,
     file_id integer not null,
+    blue_button_id text,
```

Comment:
I updated BCDA-2562 (the ticket for `r43` release) to indicate that a DB migration is required after deployment.